### ISSUE TITLE
Learning rate setting

### ISSUE THREAD

**Issue by liangjxiong** [Jeff-Zilence/TransGeo2022#37]
Hello! Your work is very outstanding. If I use 4 or 8 GPUs for training, how should I set the learning rate and what should I set it to? I hope to hear back from you. Thank you!

**Comment by Jeff-Zilence** [comment]
We did not use 4/8 GPUs. You might try the lr scaling rule for SGD. But may not generalize for Adam optimizer.

**Comment by liangjxiong** [comment]
Thank you! Can you provide a script for testing inference time?

**Comment by Jeff-Zilence** [comment]
Multi-processing evaluation can bring lots of issues. We only use one GPU for evaluation.
